base_agent/loco_mc_agent.py

```diff
@@ -118,13 +118,13 @@ def send_text_command_to_agent(sid, command):
             logical_form = {}
             status = ""
             try:
-                logical_form = dialogue_manager.get_logical_form(
-                    s=command, model=dialogue_manager.model
+                logical_form = dialogue_manager.semantic_parsing_model_wrapper.get_logical_form(
```

Comment:
Thanks for fixing this. This definitely looks like a merge conflict wasn't resolved properly after a recent refactor.